app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/convertersDTOEntityDomain/EventConverters.kt

```diff
@@ -0,0 +1,87 @@
+package com.realityexpander.tasky.agenda_feature.data.common.convertersDTOEntityDomain
+
+import com.realityexpander.tasky.agenda_feature.data.repositories.eventRepository.local.entities.EventEntity
+import com.realityexpander.tasky.agenda_feature.data.repositories.eventRepository.remote.eventApi.DTOs.EventDTO
+import com.realityexpander.tasky.agenda_feature.domain.AgendaItem
+import com.realityexpander.tasky.core.util.toUtcLong
+import com.realityexpander.tasky.core.util.toZonedDateTime
+
+// from Domain to Entity
+fun AgendaItem.Event.toEntity(): EventEntity {
+    return this.let {
+        EventEntity(
+            id = it.id,
+            title = it.title,
+            description = it.description,
+            from = it.from,
+            to = it.to,
+            remindAt = it.remindAt,
+            host = it.host,
+            isUserEventCreator = it.isUserEventCreator,
+            isGoing = it.isGoing,
+            attendeeIds = it.attendeeIds,
+            photos = it.photos,
+            deletedPhotoKeys = it.deletedPhotoKeys,
+        )
+    }
```

Comment:
Curious why you explicitly call `let` so you always have to prepend `it` here? 🤔

Reply:
Just habit. I will convert it to another context function.